src/commands/deploy/command.ts

```diff
@@ -40,6 +46,16 @@ function decideDeployType(): DeployType {
         return DeployType.NextJS;
     }
 
+    // Check if nitro.config.js exists
+    if (
+        fs.existsSync(path.join(cwd, "nitro.config.js")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.mjs")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.cjs")) ||
+        fs.existsSync(path.join(cwd, "nitro.config.ts"))
+    ) {
+        return DeployType.NitroJS;
+    }
+
```

Comment:
A configuration file is not mandatory for a nitro project, hence it can be missing and we wouldn't detect the nitro project.

So, you should also add a check if there is a `package.json` present and if it has `nitro` as a dependency.

There is an example to do this, just below for `next`.